rpc/client.py

```diff
@@ -1,4 +1,8 @@
+from os import fdopen
+from sys import argv, stdout
 import xmlrpc.client
 
+prog, subcmd, *args = argv
```

Comment:
I don't see usage of `prog`, also I don't understand how `subcmd` works

Reply:
Here is my guess without reading:

A command is like in the form `rpc subcommand param1 param2`. Because the first one (rpc) is trivial we can ignore it. The subcommand is the process we want to call in the remote.

Reply:
Yep.